kotlinKoans/src/main/java/com/epam/kotlinkoans/i_introduction/_1_Java_To_Kotlin_Converter/n01JavaToKotlinConverter.kt

```diff
@@ -13,7 +12,4 @@ fun todoTask1(collection: Collection<Int>): Nothing = TODO(
     """,
     references = { JavaCode1().task1(collection) })
 
-
-fun task1(collection: Collection<Int>): String {
-    todoTask1(collection)
-}
\ No newline at end of file
+fun task1(collection: Collection<Int>): String = JavaCode1().task1(collection)
```

Comment:
Tests failing because you implement this task wrong way. 
Task is
        Convert the Java method 'task1' of the class 'JavaCode1' into Kotlin.
        In IntelliJ IDEA or Android Studio, you can copy the Java code,
        paste it into the Kotlin file and let IDE convert it.
        Please use automatic conversion for this task only.


You convert whole class and work with it. Need to convert just method and put it instead of this one